products/openssl.md

```diff
@@ -0,0 +1,57 @@
+---
+title: OpenSSL
+layout: post
+category: framework
+iconSlug: openssl
+permalink: /openssl
+releasePolicyLink: https://www.openssl.org/policies/releasestrat.html
+eolColumn: Supported
+activeSupportColumn: false
+command: openssl version
+releaseDateColumn: true
+sortReleasesBy: releaseCycle
+releases:
+  - releaseCycle: "3.0"
+    eol: 2026-09-07
+    release: 2022-09-07
```

Comment:
@Evernow are we sure if this release date correct ? looks suspecious to me because it is in future

Reply:
202`1`

Reply:
Thanks for catching that, indeed it's supposed to be 2021, fixed with https://github.com/endoflife-date/endoflife.date/pull/1163/commits/ae01473a6e194c6c69df2ac525850d7d931f8fa0